netmanager/src/views/pages/UserList/components/UsersTable/UsersTable.js

```diff
@@ -148,225 +147,213 @@ const UsersTable = (props) => {
-                  return (
-                      <div className={classes.nameContainer}>
-                        <Avatar className={classes.avatar} src={rowData.profilePicture}>
-                          {getInitials(
-                            `${rowData.firstName + " " + rowData.lastName}`
-                          )}
-                        </Avatar>
-                        <Typography variant="body1">
-                          {" "}
-                          {rowData.firstName + " " + rowData.lastName}
-                        </Typography>
-                      </div>
-                  )
-                }
-              },
-              {
-                title: "Email",
-                field: "email",
-              },
-              {
-                title: "Username",
-                field: "userName",
-              },
-              {
-                title: "Role",
-                field: "privilege",
-              },
-              {
-                title: "Joined",
-                field: "createdAt",
-                render: (candidate) => <span>{formatDateString(candidate.createdAt)}</span>
-              },
-              {
-                title: "More Details",                
-                render: (user) => <RemoveRedEye style={{color: "green"}} onClick={() => showMoreDetails(user)} />
-              },
-              {
-                title: "Action",
-                render: (user) => {
-                  return (
-                      <div>
-                      <Button
-                        color="primary"
-                        onClick={() => showEditDialog(user)}
-                      >
-                        Update
-                      </Button>
+      <CustomMaterialTable
+        title={'Users'}
+        userPreferencePaginationKey={'users'}
+        data={users}
+        columns={[
+          {
+            title: 'Full Name',
+            render: (rowData) => {
+              return (
+                <div className={classes.nameContainer}>
+                  <Avatar className={classes.avatar} src={rowData.profilePicture}>
+                    {getInitials(`${rowData.firstName + ' ' + rowData.lastName}`)}
+                  </Avatar>
+                  <Typography variant="body1">
+                    {' '}
+                    {rowData.firstName + ' ' + rowData.lastName}
+                  </Typography>
+                </div>
+              );
+            }
+          },
+          {
+            title: 'Email',
+            field: 'email'
+          },
+          {
+            title: 'Username',
+            field: 'userName'
+          },
+          {
+            title: 'Role',
+            field: 'privilege'
+          },
+          {
+            title: 'Joined',
+            field: 'createdAt',
+            render: (candidate) => (
+              <span>{candidate.createdAt ? formatDateString(candidate.createdAt) : '---'}</span>
+            )
```

Comment:
require createdAt field before formating string

Reply:
@123MwanjeMike the createdAt field is automatically added into the db when a new candidate registers so adding the ternary operator is not needed. Just add `{formatDateString(candidate.createdAt)}` except if you found instances of empty fields?

Reply:
Some records didn't have it. Which caused @wabinyai's bug that @Baalmart shared via mail on Wednesday.

Reply:
Oh okay